python/pyspark/sql/dataframe.py

```diff
@@ -819,6 +819,29 @@ def columns(self):
         """
         return [f.name for f in self.schema.fields]
 
+    @since(2.4)
+    def colRegex(self, colName):
+        """
+        Selects column based on the column name specified as a regex and return it
```

Comment:
Nit: -> `returns`

Reply:
Unfortunately, we have the same issue in Dataset.colRegex. Please also correct that too.

Reply:
@gatorsmile Thanks for your comments. I will make the changes.